server/routes/api.js

```diff
@@ -0,0 +1,93 @@
+const express = require('express')
+const router = express.Router()
+const request = require('axios')
+
+const City = require('../models/City')
+
+function apireq (city) {
+  return request(`http://api.openweathermap.org/data/2.5/weather?q=${city}&appid=d98539d8bfaec863c4dbb2f0d8f3dc40&units=metric`)
+}
+
+router.get('/city/:cityName', async function (req, res) {
+  const city = req.params.cityName
+  const name = city.charAt(0).toUpperCase() + city.slice(1)
+  const data = await apireq(name)
+    .catch(function (err) {
+      console.error(err)
+    })
+
+  if (!data) {
+    res.status(404).end()
+    return
```

Comment:
status code 404 is when the page is not found, not when the data is empty. in this situation, you should return null or empty array.